src/GovUk.Education.ExploreEducationStatistics.Common/Extensions/EnumerableExtensions.cs

```diff
@@ -291,5 +291,15 @@ public static bool IsSameAsIgnoringOrder<T>(this IEnumerable<T> first, IEnumerab
             
             return new Tuple<T, T, T>(list[0], list[1], list[2]);
         }
+
+        public static bool ContainsAll<T>(this IEnumerable<T> source, IEnumerable<T> values)
+        {
+            if (source.IsNullOrEmpty() || values.IsNullOrEmpty())
+            {
+                return false;
+            }
+
+            return values.All(id => source.Contains(id));
+        }
```

Comment:
Discussed returning true when the right hand side is empty further with @jack-hive. I mentioned that from a mathematical point of view the empty set is a subset of any set, and that it will also be consistent with the way EF have implemented the `All` method, where it returns `true` regardless of the predicate if the argument is empty.

Reply:
Discussed and agreed :)